templates/package.json

```diff
@@ -9,6 +9,7 @@
         "verify": "node ./scripts/yamlSolver.js verify",
         "apply": "node ./scripts/yamlSolver.js apply",
         "watch": "node ./scripts/watch.js",
+        "init": "node ./scripts/yamlSolver init",
```

Comment:
Only one argument is passed here, but the YamlSolver class initialization should accept three arguments.

Reply:
Yes. The rest arguments are expected to be specify by users.